src/java.base/share/conf/security/java.security

```diff
@@ -1574,3 +1574,49 @@ jdk.tls.alpnCharset=ISO_8859_1
 # withEncryption method.
 #
 jdk.epkcs8.defaultAlgorithm=PBEWithHmacSHA256AndAES_128
+
+#
```

Comment:
Suggest combining the text and talking about both keys together since there is a lot of duplicated text.